src/Installer/redist-installer/projects/SdkResolver.csproj

```diff
@@ -7,7 +7,9 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
     <ExcludeFromSourceOnlyBuild>true</ExcludeFromSourceOnlyBuild>
-    <ExcludeFromDotNetBuild Condition="'$(DotNetBuildPass)' == ''">true</ExcludeFromDotNetBuild>
+    <!-- For product build, this project only builds in the second build pass as it depends on assets other
+         verticals that are built in the first build pass. -->
+    <ExcludeFromDotNetBuild Condition="'$(DotNetBuildPass)' != '2'">true</ExcludeFromDotNetBuild>
```

Comment:
It would be really nice to be able to auto-filter these.

Reply:
Not possible as long as we use sln/slnf files which walk all the nodes. If the new slnx format will ever get conditional support then we could switch to that and avoid walking this node and not need the condition then.